samples/tools/pbmhelper.py

```diff
@@ -0,0 +1,85 @@
+# Written by Chris Arceneaux
+# GitHub: https://github.com/carceneaux
+# Email: carceneaux@thinksis.com
+# Website: http://arsano.ninja
+#
+# This code has been released under the terms of the Apache-2.0 license
+# http://opensource.org/licenses/Apache-2.0
+
+"""
+This module implements simple helper functions for working with the
+VMware Storage Policy (pbm) API
+"""
+
+from pyVmomi import pbm, VmomiSupport
+
+
+def create_pbm_session(stub):
```

Comment:
Method name and What method returns are not matching.
Instead method should return session or pbm service instance. or make method retriev_pbm_content()

Thanks

Reply:
@pgbidkar Noted, I'll make the necessary changes to `pbmhelper.py`.

What kind of test results are you looking for in the samples? The only sample with output is `list_vdisk_snapshots.py`. The rest of the samples have error checking so the sample either succeeds or fails. Please let me know.

Thanks,
Chris

Reply:
Hi, thanks for the details. Got it. I was thinking, for every sample you print Success messages. If not I am ok without test results.

Please make the last review change and update the PR

Thanks

Reply:
@pgbidkar Requested changes have been made to `pbmhelper.py` and scripts leveraging that library.

See the screenshot below for the output of `list_vdisk_snapshots.py`.

![image](https://user-images.githubusercontent.com/17179966/58138561-5df7b280-7bfc-11e9-815d-9efe11928c21.png)